### DISCUSSION TITLE
Is it possible for markmap to support text editing?

### DISCUSSION THREAD

**Discussion by yuanlizbyy** [markmap/markmap#150] [Ideas]
It would be nice to have a simple editing function. 
Just like many mind mapping app, double click the node to edit, `tab` button to create a new node at the next level, `delete` button to delete the selected node.

**Comment by gera2ld** [comment]
Not in the foreseeable future, I think it's out of the scope of this library since everything in markmap is based on Markdown.

**Comment by IGitYou** [comment]
Just for my 2 cents if the resultant edits in a graphical markmap editor resulted in an edited Markdown file you would be cool... right? I think it would be a heavy lift but very useful. Even re-ordering or organizing in the mindmap view would be helpful for me
